src/diffusers/loaders/lora.py

```diff
@@ -333,40 +324,74 @@ def _best_guess_weight_name(
         weight_name = targeted_files[0]
         return weight_name
 
-    @classmethod
-    def _optionally_disable_offloading(cls, _pipeline):
-        """
-        Optionally removes offloading in case the pipeline has been already sequentially offloaded to CPU.
+    def fuse_lora(
+        self,
+        fuse_unet: bool = True,
+        fuse_text_encoder: bool = True,
+        lora_scale: float = 1.0,
+        safe_fusing: bool = False,
+        adapter_names: Optional[List[str]] = None,
+    ):
```

Comment:
So that the earlier public methods still work as expected. Also, showcases the power of reusability nicely.